Bio/Nexus/Nexus.py

```diff
@@ -36,7 +36,7 @@
                     'charset', 'charpartition', 'taxpartition', 'matrix',
                     'tree', 'utree', 'translate', 'codonposset', 'title']
 KNOWN_NEXUS_BLOCKS = ['trees', 'data', 'characters', 'taxa', 'sets', 'codons']
-PUNCTUATION = '()[]{}\,;:=*\'"`+-<>'
+PUNCTUATION = r'()[]{}\,;:=*\'"`+-<>'
```

Comment:
This changes the value,

```python
>>> '()[]{}\,;:=*\'"`+-<>'
'()[]{}\\,;:=*\'"`+-<>'
>>> r'()[]{}\,;:=*\'"`+-<>'
'()[]{}\\,;:=*\\\'"`+-<>'
```

Consider this example:

```
$ cat /tmp/x.py 
old = '()[]{}\,;:=*\'"`+-<>'
new = '()[]{}\\,;:=*\'"`+-<>'
assert old == new
$ flake8  --select W605 /tmp/x.py
/tmp/x.py:1:8: W605 invalid escape sequence '\,'
```

So, use this instead:

```python
PUNCTUATION = '()[]{}\\,;:=*\\\'"`+-<>'
```